android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -129,7 +133,9 @@ class RNPlayerView(context: Context, appContext: AppContext) : ExpoView(context,
             playerView?.onStop()
         }
 
-        override fun onDestroy(owner: LifecycleOwner) = dispose()
+        override fun onDestroy(owner: LifecycleOwner) {
+            playerView?.onDestroy()
+        }
```

Comment:
Is it fine to do less now on `onDestroy`? `dispose()` did more than `playerView?.onDestroy()`